include/seastar/core/seastar.hh

```diff
@@ -60,8 +60,11 @@
 #include <filesystem>
 #include <optional>
 #include <string_view>
+#include <grp.h>
```

Comment:
we can drop this. as a full definition of the `group` is not necessary, and we already have its forward declaration in place. see the other comment.